app/routes/history.py

```
@@ -0,0 +1,31 @@
+from urllib import request
```

Ruff (0.8.2)

`urllib.request` imported but unused

Remove unused import: `urllib.request`

(F401)